.github/scripts/ci-test.sh

```diff
@@ -12,7 +12,23 @@ python examples/build.py
 
 # Test with DummyVM (each test in a separate run)
 cd vmbindings/dummyvm
-for t in $(ls src/tests/ -I mod.rs | sed -n 's/\.rs$//p'); do
-    cargo test -- $t;
-done;
+for fn in $(find src/tests -name '*.rs' -a ! -name 'mod.rs'); do
+    t=$(basename -s .rs $fn)
+
+    # Get the required plans.
+    # Some tests need to be run with multiple plans because
+    # some bugs can only be reproduced in one plan but not others.
+    PLANS=$(sed -n 's;^//\s*GITHUB-CI:\s*MMTK_PLAN=\(\w\+\)\s*$;\1;p' $fn)
```

Comment:
This makes sure no regression for semispace and marksweep on those tests. But it does not prevent other plans and new plans breaking the tests. 

It would be good if we could extract plans from `PlanSelector`, like this in Python: https://github.com/mmtk/mmtk-core/blob/e85d99761935da867d766cec48e1fcdd941e8286/examples/build.py#L18
and run the tests for all the plans and all the features (some tests will be skipped if the feature they require is not enabled). Like what we do for `cargo test`.

No change is requested on this for the PR.

Reply:
MarkSweep and SemiSpaces are just two representatives for plans that has external allocators (MallocSpace) and the plans that does not have such allocators.  But it is always a good idea to test it for all plans.